Invoke-SqlCmd2/Public/Invoke-SqlCmd2.ps1

```diff
@@ -408,7 +408,7 @@ function Invoke-Sqlcmd2 {
 			}
 			
 			#Following EventHandler is used for PRINT and RAISERROR T-SQL statements. Executed when -Verbose parameter specified by caller
-			if ($PSBoundParameters.Verbose) {
+			if ($PSBoundParameters.ContainsKey('Verbose') -and $PSBoundParameters.Verbose) {
```

Comment:
Hiyo!  Didn't realize this PR was still out here, sorry!

Might be worth checking the automatic variable `$VerbosePreference` - this should capture scenarios where the user specifies `-Verbose`, and cases where the user has set verbosity or used that variable in their code.

Cheers!